templates/vue/cypress/integration/lightbox/activity.spec.js

```diff
@@ -1,25 +1,88 @@
 describe("Activity", () => {
-  it("should be able to complete an activity with an audio-based answer", () => {
+  it("should be able to switch between questions in a activity", () => {
     cy.fixture("one-node.json").as("oneNode")
     cy.setup("@oneNode")
 
     cy.getSelectedNode().then(node => {
       cy.openModal("edit", node.id)
       cy.changeMediaType("activity")
-
       const question = `What's your name?`
-
+      const placeholder = "placeholder"
+      const answer = "Tapestry"
       cy.contains(/add question/i).click()
 
       cy.contains(/question text/i).click()
       cy.focused().type(question)
-      cy.contains(/audio/i).click()
+      cy.getByTestId("question-answer-text-0").click({ force: true })
+      cy.getByTestId("question-answer-text-single-0").click({ force: true })
+      cy.getByTestId("question-answer-text-single-placeholder-0").type(placeholder)
+
+      cy.contains(/add question/i).click()
+      const question2 = `What's your favorite pet?`
+      const placeholder2 = "placeholder 2"
+      const answer2 = "Dog"
+      cy.getByTestId("question-text-1").click()
+      cy.focused().type(question2)
+      cy.getByTestId("question-answer-text-1").click({ force: true })
+      cy.getByTestId("question-answer-text-single-1").click({ force: true })
+      cy.getByTestId("question-answer-text-single-placeholder-1").type(placeholder2)
 
       cy.submitModal()
       cy.openLightbox(node.id)
+      cy.route("POST", "**/quiz*").as("submit")
```

Comment:
I thought we are no longer using *quiz* terminology? This backend route doesn't seem right.